recipes/rabbitmq.rb

```diff
@@ -57,7 +58,9 @@
     node.override["rabbitmq"]["ssl_#{item}"] = path
   end
 
-  directory File.join(ssl_directory, "client")
+  directory File.join(ssl_directory, "client") do
+    mode '0755'
```

Comment:
we might want to param this?

Reply:
is there demand for param'ing it?

Reply:
nope, I am not a maintainer on this project though I am on several other sensu projects. It's just my experience that when in doubt make it a param with the same default you planned to initially hardcode.

Reply:
if i may offer my 2c i'm not sure about adding features (or in this case params) without demand - might make the list of parameters too long and might make it harder to find the right one? what are your thoughts?

Reply:
fair enough. I know cameron is out so I am not sure when someone is gonna review this.

Reply:
do take my opinion with a grain of salt and do what you think is best for the project. and sounds good, looking forward to the review!

Reply:
It's fine for now, let someone open up a PR when they get upset that they want it more restrictive because **reasons** and wants it to be an attribute.

Reply:
Well I am now a maintainer on this so now my opinion does matter...hehe...that being said I am not gonna force it here until someone comes up with a valid reason.